Automattic/jetpack
# Admin Menu: Make Home & Stats visible in Locked Mode

For sites on 100-Year plans that have Locked Mode enabled, the Home and Stats menu item are currently removed because we disable the `edit_posts` capability.

On Simple sites, all available roles have `edit_posts` and `read` capabilities, so there is no difference in functionality in using `read` over `edit_posts` for the My Home and Stats menu items.

On WoA sites, the My Home menu item already uses the `read` capability. For stats we can use Jetpack's `view_stats` capability and filter the default options for the Stats menu to maintain access for all roles.

## Proposed changes:

* Update Stats menu item capability on WoA sites to `view_stats` to re-use Jetpack capability for that purpose.
* Add a default option callback to maintain access to Stats for all roles.
* Update My Home menu capability on Simple Sites to `read` to bring in line with Core.
* Update Stats menu item capability on Simple Sites to `read` so it doesn't get hidden when we remove the `edit_posts` capability.

### Other information:


## Does this pull request change what data or activity we track or use?



No

## Testing instructions:






* Apply the diff to your sandbox and sandbox public-api.
* Apply the diff to a WoA site using Jetpack Beta.
* Test admin menu with simple and WoA sites with admin and contributor roles.
* Make sure My Home and Stats are always accessible.
* Upgrade a test site to the 100-year plan.
* In Settings General, enable Locked Mode.
* Make sure My Home and Stats are always accessible.
